src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -122,6 +122,17 @@ export const SubmissionHistory = () => {
                 paddingTop: '1em'
               }),
               columns: [
+                {
```

Comment:
We should hide the Actions column behind a feature flag. We don't want users to use this functionality until everything in Abort epic is complete.